build/dockerfiles/linux-libc-ubi8.Dockerfile

```diff
@@ -25,8 +25,8 @@ ENV DISABLE_V8_COMPILE_CACHE=1
 
 # Install libsecret-devel on s390x and ppc64le for keytar build (binary included in npm package for x86)
 RUN { if [[ $(uname -m) == "s390x" ]]; then LIBSECRET="\
-      https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-0.20.4-2.fc34.s390x.rpm \
-      https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-devel-0.20.4-2.fc34.s390x.rpm"; \
+      https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi8/8/s390x/baseos/os/Packages/l/libsecret-0.18.6-1.el8.s390x.rpm \
+      https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi8/8/s390x/baseos/os/Packages/l/libsecret-devel-0.18.6-1.el8.s390x.rpm"; \
```

Comment:
I've added a new reference for the `libsecret` and `libsecret-devel` packages.
Build passed that step successfully - thanks a lot Sam Ding for testing it on the `s390x` platform!

Unfortunately, there is another error, related to `xkbfile` package:

<img width="1142" alt="image" src="https://github.com/user-attachments/assets/6fd27d7a-cd71-4303-88bf-30cb6b2042a0">